base-local/Dockerfile

```diff
@@ -65,7 +65,7 @@ FROM base AS lcpserver
 
 COPY --from=builder /go/bin/lcpserver /go/bin/lcpserver
 ENV READIUM_LCPSERVER_CONFIG=/etc/readium/config.yaml
-CMD ["sh", "-c", "envsubst < /etc/readium/config.yaml > /etc/readium/config.yaml; /go/bin/lcpserver"]
+CMD ["sh", "-c", "envsubst < /etc/readium/config.yaml > /tmp/config.yaml && mv /tmp/config.yaml /etc/readium/config.yaml; /go/bin/lcpserver"]
```

Comment:
I don't understand this change. Line 32 of the original Dockerfile copies the contents of the local `etc` directory to the build's `/etc/readium` directory, so the config.yaml file is already present in the build.

Reply:
`envsubst < /etc/readium/config.yaml > /etc/readium/config.yaml` is used for substituting environment variables in `/etc/readium/config.yaml`:
- the first redirection operator sends the raw content of `/etc/readium/config.yaml` to `envsubst`
- the second redirection operator sends the substituted content back to `/etc/readium/config.yaml`

However, redirect `> /etc/readium/config.yaml` [has effect of emptying the file](https://stackoverflow.com/questions/35078753/can-envsubst-not-do-in-place-substitution) and we end up with an empty `/etc/readium/config.yaml` (at least in MacOS).

I changed the command to use a temporary file:
- first, `envsubst` writes the substituted content to `/tmp/config.yaml`
- secondly, we rename `/tmp/config.yaml` to `/etc/readium/config.yaml`

However, it also led me to think about possible race conditions when all three containers (`lcpserver`, `lsdserver` and `testfrontend` start at the same time) and some of them might end up with a corrupted version of `/etc/readium/config.yaml`

Reply:
I see now. I knew the function of the substitution. But I don't have a Mac, so the error you experienced I haven't seen. Good catch.